packages/cozy-harvest-lib/src/helpers/fields.js

```diff
@@ -46,6 +46,8 @@ export const sanitizeSelectProps = props => {
   delete sanitized.onKeyUp
   // Not applicable on Selects
   delete sanitized.autoComplete
+  // Only for inputs
+  delete sanitized.inputRef
```

Comment:
```
const sanitized = omit(props, 'onKeyUp', 'inputRef', 'autoComplete')
```

Reply:
In the description of the function, it's written react-select and then Field component ? I guess it should read "Prepare props to pass to Field component" ? Could we use Field.propTypes to only pass relevant properties ?

```
const propTypes = pick(props, Object.keys(Field.propTypes))
```

?

Reply:
Oh nice, I love this approach, thanks !

Reply:
Actually, what we do is:
Renders Field, which renders SelectBox, which renders ReactSelect.

Removing unusable props should be done in `<Field />`.

Reply:
So, `<ReactSelect />` component does not expose propTypes, so we cannot consider them in `<SelectBox />` component, and in `<Field />` component neither.

The best solution I found to clean props we pass to `<SelectBox />` is here. https://github.com/cozy/cozy-ui/pull/863

Reply:
I removed the `inputRef` deletion and will remote the other line in a dedicated PR.